testing.rst

```diff
@@ -26,10 +33,36 @@ you wish to test with.
 
 .. code-block:: shell
 
-    RE_JOB_ACTION=r12.2.8_to_r14.6.0_leap ./run-tests.sh
+    export RE_JOB_ACTION=r12.2.8_to_r14.6.0_leap
+    ./run-tests.sh
 
 
 When you executing the `run-tests.sh` script a full AIO will be
 built and then the upgrade tools executed against it. This will
 allow for the rapid testing and proto-typing within a localized
 environment.
+
+======================================
+Creating a test MNAIO (Multi Node AIO)
+======================================
+
+Leapfrog MNAIO Test Process
+---------------------------
+
+Build an OnMetal server using the flavor onmetal-io1 and running
+Ubuntu 16.04 LTS.
+
+You'll need to set the RE_JOB_IMAGE to xenial_mnaio in order to
+trigger the mnaio code path.  Then set the action to set the
+leap you want to test.
+
+.. code-block:: shell
+
+    export RE_JOB_IMAGE=xenial_mnaio
```

Comment:
Does this tell o-a-ops to spin it's vms as xenial vms?
If so, shouldn't this be trusy_mnaio?
If not, maybe some clarifying explanation.

Reply:
Yeah, my bad... I've been in xenial mode this week, I'll correct.

Reply:
Eh, actually that's correct... we're using a Xenial Host now for MNAIO testing, the image that it pulls down will be Trusty though.

Reply:
This is where you can tune the gating script to use Trusty or Xenial as the VM.  Right now the assumption is all leapfrogs start from Trusty: 

https://github.com/rcbops/rpc-upgrades/blob/master/tests/create-mnaio.sh#L39

Reply:
Cool. Thanks for the clarification. We might want to just dump that comment/blurb/knowledge into this doc so the next person reading it knows that too.

Reply:
Added a quick blurb about it.